packages/k6-generator/src/index.js

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const replace = require('replace-in-file');
+const { StringPrompt } = require('enquirer');
+
+const templateFile = fs.readFileSync('./template.js');
+
+let [customImport, customMetrics, writeVuObj, uniqueObj, mainContent] = ['', '', '', '', ''];
+
+module.exports.run = async () => {
+  const methodsPath = await new StringPrompt({
+    initial: './methods/expressTemplate.json',
+    message: 'Confirm the path to the methods JSON file:',
+  }).run();
+
+  if (
+    !fs.existsSync(`${__dirname}${methodsPath.slice(methodsPath.indexOf('/'))}`) ||
+    methodsPath.slice(-5) !== '.json'
+  ) {
+    console.log(`${methodsPath} is not a valid path`);
+    process.exit();
+  }
+
+  const methods = require(`${__dirname}${methodsPath.slice(methodsPath.indexOf('/'))}`);
+
+  const newFile = await new StringPrompt({
+    initial: 'tests/k6/scripts/alpha.js',
```

Comment:
Should we default this to something from the input file? If the input file is `expressTemplate.json`, we can default the filename to something like `expressTemplate.k6.js`